andes/models/group.py

```diff
@@ -243,31 +243,41 @@ def set(self, src: str, idx, attr, value):
 
         return True
 
-    def find_idx(self, keys, values, allow_none=False, default=None):
+    def find_idx(self, keys, values, allow_none=False, default=None,
+                 no_flatten=False):
```

Comment:
`no_flatten` here introduces more ambiguity. Remove it since we don't actually need it.